archive/zip.ksy

```diff
@@ -334,6 +334,8 @@ enums:
     18: ibm_terse
     19: ibm_lz77_z
     98: ppmd
+  # doc-ref: https://github.com/LuaDist/zip/blob/b710806/proginfo/extrafld.txt
+  # doc-refs in enums are not currently supported. Note to self: Update when it is supported.
```

Comment:
Create/find an issue about it and use `-affected-by`

Reply:
@KOLANICH https://github.com/kaitai-io/kaitai_struct/issues/358, see https://github.com/kaitai-io/kaitai_struct_formats/pull/480#discussion_r616613528

Reply:
And what do I do to use this ``-affected-by``?